src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -2013,43 +2011,84 @@ tools to limit workflow activity.
 Runahead Limiting
 ^^^^^^^^^^^^^^^^^
 
-Runahead limiting, controlled by :cylc:conf:`[scheduling]runahead limit`,
-prevents the fastest tasks in a workflow from getting too far
-ahead (with respect to cycle point) of the slowest ones.
+The scheduler runahead limit determines how many consecutive cycle points can
```

Comment:
Do the cycle points have to be consecutive to be affected by the runahead limit?

Reply:
Yes, the runahead limit is computed by counting consecutive cycle points. (We've considered using just active cycle points, but don't have that at the moment).